### PR TITLE
merge branch S2X to develop

### PR DESCRIPTION
**Pull request by CescWang1991**

### CODE REVIEW

**Comment by CescWang1991** [comment]
Append package net.sansa_stack.query.spark.graph to SANSA_Query develop branch

**Comment by CescWang1991** [comment]
Add resources and append unit tests for graph queries.

File: sansa-query-spark/pom.xml [hunk 1]
```diff
@@ -22,6 +22,13 @@
 			<artifactId>sansa-rdf-spark${scala.version.suffix}</artifactId>
 		</dependency>
 
+		<!-- SANSA RDF -->
```
Comment (GezimSejdiu) [inline sansa-query-spark/pom.xml]: We are going to deprecate this reader on a new version of SANSA and for that there is no need to use this dependecny anymore!

File: sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/graph/jena/SparqlParser.scala [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    this("", op)
+  }
+
+  private val elementTriples = ArrayBuffer[Triple]()
+  private val ops = new mutable.Queue[Ops]()
+
+  OpWalker.walk(op, this)
+
+  override def visit(opBGP: OpBGP): Unit = {
+    println("opBGP: "+opBGP)
```
Comment (GezimSejdiu) [inline sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/graph/jena/SparqlParser.scala]: Remove println statements!

File: sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/graph/jena/SparqlParser.scala [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    val triples = opBGP.getPattern.toList
+    for (triple <- triples) {
+      elementTriples += triple
+    }
+    if(ops.isEmpty){ ops.enqueue(new PatternBgp(elementTriples.toIterator)) }
+  }
+
+  override def visit(opDistinct: OpDistinct): Unit = {
+    println("opDistinct: "+opDistinct)
```
Comment (GezimSejdiu) [inline sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/graph/jena/SparqlParser.scala]: Remove println statements! -- Same as above -- remove everywhere!

**Pull request merged**